### ISSUE TITLE
[Screen Reader-MCIO DCX-'Site Selection' page-Name filter]: There is no visual label for the 'search' field present under 'Name' filter pane and also the placeholder text is not descriptive.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7242]
**Description**:  Visual users with limited cognitive abilities will be affected as they cannot easily understand the field if there is no visual label for the field. Also, screen reader users will be affected if the placeholder text is not descriptive enough.

**Environment (OS, Application, Versions)**:
**Test Environment**:
OS Build: Version 21H2(22000.856)
Browser: Chrome Version 105.0.1343.25 (Official build) (64-bit) 
Screen reader: Non-Visual Desktop Access (NVDA) (2022.2)

**Repro Steps**: 
1. Open "https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering"
2. Grids excel style filtering pen page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Press ctrl + shift + L, the context menu opens.
6. With 9-tab press, reach to the search box of the opened pane.
7. Verify the issue that label is provided for the search box, or the place holder text is descriptive enough.

**Actual Result**: There is no visual label for the 'search' field present under the 'Name' filter pane and also the placeholder text is not descriptive.

**Expected Result**: There should be a visual label for the 'search' field present under the 'Name' filter pane and also the placeholder text should be descriptive.

**Additional Information**: Same Issue Reproes in the same dialog present throughout the application.

**Attachment**: "https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-deenthakur_microsoft_com/Eas0J5n97zBKj1HyztIojRoB0MJpMKN2eha1joPRknc6nA?e=MU9tVH"

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
